instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/utils/StringExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package com.datadog.android.sdk.utils
+
+internal fun String.isTracesUrl(): Boolean {
+    return this.endsWith("traces")
+}
+
+internal fun String.isLogsUrl(): Boolean {
+    return this.matches(Regex("(.*)logs(.*)"))
```

Comment:
why not `this.endsWith("logs")` ?

Reply:
There's more after "logs" in the url for logs.

Reply:
Maybe you could have something more specififc like `(.*)/logs/(.*)` ?